spacepy/pybats/cimi.py

```diff
@@ -0,0 +1,343 @@
+#!/usr/bin/env python
+'''
+PyBats submodule for handling input/output for the CIMI Model
+, one of the choices for the IM module in the SWMF.
+'''
+
+# Global imports:
+import numpy as np
+import datetime as dt
+import spacepy.plot.apionly
+from spacepy.plot import set_target
+from spacepy.pybats import PbData
+from spacepy.datamodel import dmarray
+
+
+
+
+class CimiEq(PbData):
+    '''
+    Class for loading a CIMIeq.out file.
+    At instantiation time, user may wish to set the start date and time of 
+    the simulation using the starttime kwarg.  If not given, start time
+    will default to Jan. 1st, 2000, 00:00UT.
+    '''
```

Comment:
The arguments to the initializer should be documented in here...see [257](https://peps.python.org/pep-0257/) and an [example of formatting](https://github.com/spacepy/spacepy/blob/a58579f79a8b78729a1209cc399f301524cd0b1c/spacepy/pybats/bats.py#L23)